src/utils/run-build.ts

```diff
@@ -140,7 +145,7 @@ export const runNetlifyBuild = async ({
     if (!options.dir && netlifyConfig?.build?.publish) {
       settingsOverrides.dist = netlifyConfig.build.publish
     }
-    await devCommand(settingsOverrides)
+    await devCommand({ netlifyConfig })
```

Comment:
what happened to `settingsOverrides`?

Reply:
Thanks, my stash went crazy and not noticed it wasn't added to commit. Fixed